tests/wpt/meta/compression/compression-bad-chunks.tentative.any.js.ini

```diff
@@ -0,0 +1,62 @@
+[compression-bad-chunks.tentative.https.any.shadowrealm-in-audioworklet.html]
+  expected: ERROR
+
+[compression-bad-chunks.tentative.https.any.shadowrealm-in-serviceworker.html]
+  expected: ERROR
+
+[compression-bad-chunks.tentative.any.shadowrealm-in-dedicatedworker.html]
+  expected: ERROR
+
+[compression-bad-chunks.tentative.any.worker.html]
+  [chunk of type SharedArrayBuffer should error the stream for gzip]
```

Comment:
Above I see this implemented, yet the tests are failing. Do you know what we are missing here?

Reply:
Correct.